package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "postcss-sorting",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

```suggestion
  "version": "5.0.0",
```

Please remove this change. It will mess with my publishing tool.